ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -850,5 +850,11 @@ private MDRDetails ExtractMdrDetailsAsync(MigrationDbNotification rawNotificatio
 
             return mdr;
         }
+
+        private DenotificationReason GetDenotificationReason(MigrationDbNotification notification)
+        {
+            var reason = (DenotificationReason[])Enum.GetValues(typeof(DenotificationReason));
+            return reason.SingleOrDefault(r => r.GetDisplayName().ToLower() == notification.DenotificationReason.ToLower());
```

Comment:
By using the display name here, we're tying the display text that we use in NTBS to that which currently exists in ETS. A more flexible approach would be to use another attribute to store the ETS text for the reasons and match against that here. The downside would be that it would mean a lot of duplicate attributes.

So maybe for now we should just put a comment on the `DenotificationReason` enum, to make sure that no one accidentally changes those values without realising the consequences.

Reply:
I've added a comment. SingleOrDefault was returning 'Not selected' as the default (as that's the enum's default (0)) but actually we want the Other as default, and I've made it so it only set's the "other description" in this case.